lbrynet/conf.py

```diff
@@ -562,6 +578,9 @@ class Config(CLIConfig):
     streaming_get = Toggle("Enable the /get endpoint for the streaming media server. "
                            "Disable to prevent new streams from being added.", True)
 
+    coin_selection_strategy = OneOfString(STRATEGIES, "Strategy to use when selecting UTXOs for a transaction",
+                                          "branch_and_bound")
```

Comment:
This default would only run `branch_and_bound` and not the others. I think default should be `None` or `"auto"` which lets torba decide which strategy is best (currently doing the 3 strategies it does in specific order).